native/meeseeks_html5ever_nif/src/lib.rs

```diff
@@ -175,23 +169,18 @@ fn parse<'a>(parser_type: ParserType, env: Env<'a>, args: &[Term<'a>]) -> NifRes
     }
 }
 
-fn parse_html<'a>(env: Env<'a>, args: &[Term<'a>]) -> NifResult<Term<'a>> {
-    parse(ParserType::HtmlDocument, env, args)
+#[rustler::nif(schedule = "DirtyCpu")]
+fn parse_html<'a>(env: Env<'a>, document: Binary) -> NifResult<Term<'a>> {
+    parse(ParserType::HtmlDocument, env, document)
 }
 
-fn parse_xml<'a>(env: Env<'a>, args: &[Term<'a>]) -> NifResult<Term<'a>> {
-    parse(ParserType::XmlDocument, env, args)
+#[rustler::nif(schedule = "DirtyCpu")]
+fn parse_xml<'a>(env: Env<'a>, document: Binary) -> NifResult<Term<'a>> {
+    parse(ParserType::XmlDocument, env, document)
 }
 
-rustler_export_nifs!(
-    "Elixir.MeeseeksHtml5ever.Native",
-    [
-        ("parse_html", 1, parse_html, SchedulerFlags::DirtyCpu),
-        ("parse_xml", 1, parse_xml, SchedulerFlags::DirtyCpu),
-    ],
-    Some(on_load)
-);
+rustler::init!("Elixir.MeeseeksHtml5ever.Native", [parse_html, parse_xml], load = load);
 
-fn on_load<'a>(_env: Env<'a>, _load_info: Term<'a>) -> bool {
+fn load(_env: Env, _load_info: Term) -> bool {
```

Comment:
Curious what prompted the change from `on_load` to `load`?

Reply:
Ah this was honestly only to align with what rustler.new would generate if ran today, it's not a required change